City-of-Helsinki/drupal-helfi-platform-config
# UHF-679: Export translations via drush import:locale

To test this:

- Make sure drupal/helfi_api_base is up to date
- Run `drush helfi:locale-import helfi_platform_config`
- Open `/admin/config/regional/translate`, search for `Test string` and verify it has translations
